plugins/autoclothing.cpp

```diff
@@ -459,7 +474,10 @@ command_result autoclothing(color_ostream &out, vector<string> &parameters)
         if (settingSize) {
             if (newRequirement.needed_per_citizen == 0) {
                 clothingOrders.erase(clothingOrders.begin() + i);
-                out << "Unset " << parameters[0] << " " << parameters[1] << endl;
+                    if (parameters[0] == "clear")
```

Comment:
I don't like this sort of downstream dependency on the specific command issued is bad design, but this is an architectural problem with the way this plugin was implemented in the first place and isn't a problem with this PR so I'm not going to block this PR on it